content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/delivery_three_steps.rst

```diff
@@ -118,18 +100,17 @@ moves to the :guilabel:`WH/Output` location. Then, the status of the document wi
 Process a delivery
 ------------------
 
-The delivery order will be ready to be processed once the packing is completed, and can be found in
-the :menuselection:`Inventory` application :guilabel:`Overview` dashboard. Click the :guilabel:`1 To
-Process` smart button in the :guilabel:`Delivery Orders` kanban card.
+Once the packing order has been validated, the delivery order is ready to process. Navigate back to
+the original sales order to process the delivery by going to :menuselection:`Sales app`, and
+selecting the sales order created previously.
```

Comment:
In a note admonition block, we can mention that delivery orders can also be accessed by going to Inventory > Operations > Deliveries.

Reply:
Also, I'm not sure if navigating back to the sales order is the go-to instruction here. To understand your perspective here, what was your reasoning for going back to the SO? I understand that doing this completes the sales flow, so the customer can invoice the client next. However, since this doc is in inventory, shouldn't we just go to Inventory > Operations > Delivery Orders, and process the delivery order? 

If you agree with this logic, I'd put the part about going back to the sales app in a note block, instead.

Reply:
I added a tip admonition to inform the reader that they can also access it that way. But, I'm also keeping the original instruction as I had it, too. The logic of going to the original SO was to show how the Delivery smart button updates with all the transfers as the process is completed, so that all transfers are traceable from one place. Plus, the SO needs to be closed out and invoiced once the delivery is completed.

If I get additional feedback concerning this, however, I will consider changing it.